Packs/CommonScripts/Scripts/Set/README.md

```diff
@@ -132,3 +132,14 @@ There are no outputs for this script.
 ## Human Readable Output
 
 >Key Data(val.ID == obj.ID) set
+
+## Known Limitations
+The script has some limitations with appending to the context in different cases and formats:
+- When *append* is set to `false` - the script will not support [dt](https://xsoar.pan.dev/docs/integrations/dt)operations given as part of the *key* argument.
+- When **Set** is called from within another script - if *append* is set tot `true` it is necessary to manually
+return the results given by the `demisto.executeCommand`, for the context update to take place.
+For example:
+    ```python
+    res = demisto.executeCommand('Set', {'append': 'true', 'key': 'MyKey', 'value': 'MyValue'})
```

Comment:
Sorry I forgot we encourage `execute_command` on top of `demisto.executeCommand` (it does the error handling for you) 
```suggestion
    res = execute_command('Set', {'append': 'true', 'key': 'MyKey', 'value': 'MyValue'})
```